[2024-04-22T18:16:53.479+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-22T18:16:53.528+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2024-04-16T00:00:00+00:00 [queued]>
[2024-04-22T18:16:53.537+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2024-04-16T00:00:00+00:00 [queued]>
[2024-04-22T18:16:53.539+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-22T18:16:53.554+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): task1> on 2024-04-16 00:00:00+00:00
[2024-04-22T18:16:53.583+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1081) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-22T18:16:53.585+0000] {standard_task_runner.py:63} INFO - Started process 1085 to run task
[2024-04-22T18:16:53.584+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_backfill_v02', 'task1', 'backfill__2024-04-16T00:00:00+00:00', '--job-id', '396', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmpq68e_nrq']
[2024-04-22T18:16:53.587+0000] {standard_task_runner.py:91} INFO - Job 396: Subtask task1
[2024-04-22T18:16:53.666+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2024-04-16T00:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-22T18:16:53.778+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='dag_with_catchup_backfill_v02' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-04-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-04-16T00:00:00+00:00'
[2024-04-22T18:16:53.779+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-22T18:16:53.781+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-22T18:16:53.782+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-04-22T18:16:53.788+0000] {subprocess.py:86} INFO - Output:
[2024-04-22T18:16:53.789+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-04-22T18:16:53.791+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-22T18:16:53.793+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-22T18:16:53.817+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v02, task_id=task1, execution_date=20240416T000000, start_date=20240422T181653, end_date=20240422T181653
[2024-04-22T18:16:53.862+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-22T18:16:53.887+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-22T18:16:53.888+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
